Project.toml

```diff
@@ -0,0 +1,18 @@
+name = "Highlights"
+uuid = "eafb193a-b7ab-5a9e-9068-77385905fa72"
+version = "0.4"
```

Comment:
How does registrator handle the missing patch version number? Should this be updated to be `0.4.0`?

Reply:
Yes

Reply:
Done